tensorboard/notebook.py

```diff
@@ -395,11 +395,31 @@ def _display_colab(port, height, display_handle):
 
 def _display_ipython(port, height, display_handle):
   import IPython.display
-  iframe = IPython.display.IFrame(
-      src="http://localhost:%d" % port,
-      height=height,
-      width="100%",
-  )
+  import cgi
+  import json
+  import random
+
+  frame_id = "tensorboard-frame-{:08x}".format(random.randrange(0, 1 << 64))
```

Comment:
nit: `random.randrange(0, 1 << 64)` can be `random.getrandbits(64)`,
which is a bit clearer.